src/steg86/binary.rs

```diff
@@ -151,6 +157,12 @@ static SEMANTIC_PAIRS: &[(Code, Code)] = &[
     (Code::Cmp_rm16_r16, Code::Cmp_r16_rm16),
     (Code::Cmp_rm32_r32, Code::Cmp_r32_rm32),
     (Code::Cmp_rm64_r64, Code::Cmp_r64_rm64),
+
+    // TEST
+    (Code::Test_rm8_r8, Code::Test_rm8_r8),
+    (Code::Test_rm16_r16, Code::Test_rm16_r16),
+    (Code::Test_rm32_r32, Code::Test_rm32_r32),
+    (Code::Test_rm64_r64, Code::Test_rm64_r64),
```

Comment:
Perhaps we could drop this, since `TEST` doesn't form pairs the way that our other instructions do? The `unwrap` we perform below should still be infallible since we'll still only do it for the non-commutative case.

Reply:
Makes sense, I didn't do that to make the least amount of changes, but changed now